### PR TITLE
Add example for Kubernetes deployment

### PR DESCRIPTION
**Pull request by thebillkidy**
# Description

Adding example for Kubernetes deployment

## Issue reference

#30 as well as #27 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [x] Extended the documentation

### CODE REVIEW

**Comment by youngbupark** [comment]
@thebillkidy Sorry for the late response. Thanks for your contribution. In 0.7.0, we changed gRPC proto files a lot and added the script to generate proto stub and client codes. So the current code will not work once we release 0.7.0 release. 

Here is my comments on the PR:
1. Please wait to merge this pr until 0.7.0 dapr is released
2. Use `dapr-client` instead of generating proto client codes while building docker image
3. @pruthvidhodda already merged the proto client [generation script](https://github.com/dapr/python-sdk/blob/master/protobuf.sh)

**Comment by thebillkidy** [comment]
> @thebillkidy Sorry for the late response. Thanks for your contribution. In 0.7.0, we changed gRPC proto files a lot and added the script to generate proto stub and client codes. So the current code will not work once we release 0.7.0 release.
> 
> Here is my comments on the PR:
> 
> 1. Please wait to merge this pr until 0.7.0 dapr is released
> 2. Use `dapr-client` instead of generating proto client codes while building docker image
> 3. @pruthvidhodda already merged the proto client [generation script](https://github.com/dapr/python-sdk/blob/master/protobuf.sh)

Perfect, thanks a lot! Will wait until release and then update :)

**Comment by yaron2** [comment]
Would love to see this going forward now.

**Comment by youngbupark** [comment]
@shalabhms Seems like we didn't publish the latest dapr-client sdk. Can you please publish it?

JFYI - I will set up CI/CD to publish it automatically in this milstone. 

cc/ @amanbha

**Comment by thebillkidy** [comment]
I blocked some time for this tomorrow, will finalize it then if the dapr-client sdk was pushed correctly :)

**Comment by amanbha** [comment]
> @shalabhms Seems like we didn't publish the latest dapr-client sdk. Can you please publish it?
> 
> JFYI - I will set up CI/CD to publish it automatically in this milstone.
> 
> cc/ @amanbha

I have uploaded the new python package

**Comment by youngbupark** [comment]
> > @shalabhms Seems like we didn't publish the latest dapr-client sdk. Can you please publish it?
> > JFYI - I will set up CI/CD to publish it automatically in this milstone.
> > cc/ @amanbha
> 
> I have uploaded the new python package

@amanbha Thanks! :)

**Comment by thebillkidy** [comment]
@youngbupark example updated for Dapr v0.7.1 😊

P.S. Thanks a lot for updating the other examples as well!

File: examples/kubernetes/invoke-simple/Proto/dapr.proto [hunk 1]
```diff
@@ -0,0 +1,120 @@
+syntax = "proto3";
+
+package dapr;
+
+import "google/protobuf/any.proto";
```
Comment (youngbupark) [inline examples/kubernetes/invoke-simple/Proto/dapr.proto:5]: Do we still need this proto file?

Comment (thebillkidy) [reply examples/kubernetes/invoke-simple/Proto/dapr.proto]: Nope, removed it :)

File: examples/kubernetes/invoke-simple/Scripts/generate-proto.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+if [ -z $1 -o -z $2 ]; then
+  echo "Usage: ./generate-proto.sh <LOCATION_PROTO> <OUTPUT_FOLDER>"
```
Comment (youngbupark) [inline examples/kubernetes/invoke-simple/Scripts/generate-proto.sh:2]: Nit: we can remove this script now because there is the script in root.

**Pull request merged**